src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+    },
+
+    subtract(value) {
+      this.result -= value;
+    },
+
+    multiply(value) {
+      this.result *= value;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        this.result /= value;
+      }
```

Comment:
While the check for division by zero is necessary, it would be better to throw an error when an attempt to divide by zero is made. This would prevent silent failures and make the behavior of the calculator more predictable for the user. Consider throwing an error with a descriptive message.